tennis-scheduler-web-app/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_TENNIS_COURTS=http://localhost:8081/tennis-courts/
```

Comment:
Its better in .env only to be http://localhost:8081/ and later in files for sending request you add what you need. With that everyone could use it for their requests